evalml/automl/automl_search.py

```diff
@@ -94,6 +94,8 @@ def __init__(self,
                  problem_configuration=None,
                  train_best_pipeline=True,
                  pipeline_parameters=None,
+                 sampler=None,
+                 categorical_columns=None,
```

Comment:
@bchen1116 could you please explain why this input is necessary? If I follow right, we should be using woodwork to get this information.

Reply:
Yea I agree that it'd be nice if we could avoid adding this argument to automl search.